src/controller/tests/TestEventChunking.cpp

```diff
@@ -542,10 +542,10 @@ nlTestSuite sSuite =
 
 } // namespace
 
-int TestReadChunkingTests()
+int TestEventChunkingTests()
```

Comment:
What was wrong with this test ? duplicate names ?

Reply:
Yes, a mismatch between the file name and test and TestEventChunkingTests already exists.